designer/fitting/dwipi.py

```diff
@@ -29,8 +29,9 @@ def __init__(self, imPath, nthreads=-1):
             assert isinstance(imPath, object)
             self.hdr = nib.load(imPath)
             self.img = np.array(self.hdr.dataobj)
-            nanidx = np.isnan(self.img)
-            self.img[nanidx] = minZero
+            zeroIdx = np.logical_and(np.isnan(self.img),
```

Comment:
I think it should be named something other than zeroIdx, maybe truncateIdx?

Reply:
Applied this change. Squash and merge.